clang/test/CodeGen/call-graph-section-2.cpp

```diff
@@ -0,0 +1,95 @@
+// Tests that we assign appropriate identifiers to indirect calls and targets
```

Comment:
can we get more descriptive file names than `call-graph-section-1` `call-graph-section-2`? For instance, its nice if you can tell which properties are being checked in each file based on its name.